langkit/envs.py

```diff
@@ -48,9 +48,9 @@ def add_env(no_parent: bool = False,
     :param no_parent: If passed, the new env will be created with no parent
         env.
     :param transitive_parent: TODO.
-    :param names: Optional list of names for the created environment. If not
-        passed or if this is an empty array, the created environment is not
-        named.
+    :param names: Optional array of names (symbols) for the created
```

Comment:
I don't understand, why is this still an array ?

Reply:
Because an env spec can create an environment and associate multiple env names to it (it used to be an array of array of symbols).